### ISSUE TITLE
angular generate-proxy lack of $ syntax

### ISSUE THREAD

**Issue by PooriaShariatzadeh** [abpframework/abp#7160]
### GitHub Issues

in angular ui after we generate proxy some methods for passing params need $ syntax.

getServiceByServiceUniqueId = (ServiceUniqueId: string) =>
    this.restService.request<any, Service>({
      method: 'GET',
      // url: `/api/app/tourist-panel/service/{ServiceUniqueId}`,  this is generated by proxy
      url: `/api/app/tourist-panel/service/${ServiceUniqueId}`,// it must be like this to work 
    },
    { apiName: this.apiName });

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

**Comment by maliming** [comment]
* Your **ABP CLI version**.
* Steps needed to **reproduce** the problem.

**Comment by PooriaShariatzadeh** [comment]
ABP CLI version: 4.1.0
1-serverside is run ok 
2-abp generate-proxy
3-generated proxy has not connected with API on some proxy's
```
`getServiceByServiceUniqueId = (ServiceUniqueId: string) =>
    this.restService.request<any, Service>({
      method: 'GET',
      url: `/api/app/tourist-panel/service/{ServiceUniqueId}`,
    },
    { apiName: this.apiName });`
```

edited to 

```
`getServiceByServiceUniqueId = (ServiceUniqueId: string) =>
    this.restService.request<any, Service>({
      method: 'GET',
      url: `/api/app/tourist-panel/service/${ServiceUniqueId}`,
    },
    { apiName: this.apiName });`
```
just add $ to `{ServiceUniqueId}` like `${ServiceUniqueId}`
after editing its work like a charm

**Comment by maliming** [comment]
hi @PooriaShariatzadeh 

Can you provide a project that reproduce the problem?

**Comment by PooriaShariatzadeh** [comment]
it's a simple problem I don't get why you want the whole project?

**Comment by PooriaShariatzadeh** [comment]
and I solved it already. I just don't want every time I do generate-proxy in angular side open proxies and edit it
if you just assigned it to someone who knew the angular side he/she can fix it eminently

**Comment by maliming** [comment]
You can create a project with CLI then try to reproduce the problem.

**Comment by PooriaShariatzadeh** [comment]
https://filebin.net/4i92kttil0gr5fdg/_problem.rar?t=i0uzz5rm

problem is in : angular\src\app\proxy\problem-demo.service.ts

**Comment by armanozak** [comment]
This will be available as of v4.1

**Comment by PooriaShariatzadeh** [comment]
@armanozak  thanx for your reply and commitment
